envoy/stream_info/stream_info.h

```diff
@@ -651,6 +651,16 @@ class StreamInfo {
     downstream_info.setUpstreamBytesMeter(upstream_info.getUpstreamBytesMeter());
     upstream_info.setDownstreamBytesMeter(downstream_info.getDownstreamBytesMeter());
   }
+
+  /**
+   * @return Connection ja3 fingerprint hash of the downstream connection, if any.
+   */
+  virtual const std::string& ja3Hash() const PURE;
```

Comment:
I think the changes to this interface can be reverted; you can already access this via (improperly named due to a rename at some point) `virtual const Network::ConnectionInfoProvider& downstreamAddressProvider() const PURE;`

Reply:
Reverted in commit 34ba6e20b006a8f73181b52d8505965d0327fc31.